pep-0265.txt

```diff
@@ -5,202 +5,213 @@ Last-Modified: $Date$
+order::
 
-        (3) items(values_first=0)
+    (3) items(values_first=0)
 
-    Again, this is fully backwards-compatible.  It does less work than
-    the others, but it at least eases the most complicated/tricky part
-    of the sort-by-value problem: inverting the order of item tuples.
-    Using this is very simple:
+Again, this is fully backwards-compatible.  It does less work than
+the others, but it at least eases the most complicated/tricky part
+of the sort-by-value problem: inverting the order of item tuples.
+Using this is very simple::
 
-        items = d.items(1)
-        items.sort()
-        items.reverse()         # (if desired)
+    items = d.items(1)
+    items.sort()
+    items.reverse()         # (if desired)
 
-    The primary drawback of the preceding three approaches is the
-    additional overhead for the parameter-less "items()" case, due to
-    having to process default parameters.  (However, if one assumes
-    that items() gets used primarily for creating sort-by-value lists,
-    this is not really a drawback in practice.)
+The primary drawback of the preceding three approaches is the
+additional overhead for the parameter-less ``items()`` case, due to
+having to process default parameters.  (However, if one assumes
+that ``items()`` gets used primarily for creating sort-by-value lists,
+this is not really a drawback in practice.)
 
-    Alternatively, we might add a new dictionary method which somehow
-    embodies "sorting".  This approach offers two advantages.  First,
-    it avoids adding overhead to the items() method.  Second, it is
-    perhaps more accessible to newbies: when they go looking for a
-    method for sorting dictionaries, they hopefully run into this one,
-    and they will not have to understand the finer points of tuple
-    inversion and list sorting to achieve sort-by-value.
+Alternatively, we might add a new dictionary method which somehow
+embodies "sorting".  This approach offers two advantages.  First,
+it avoids adding overhead to the ``items()`` method.  Second, it is
+perhaps more accessible to newbies: when they go looking for a
+method for sorting dictionaries, they hopefully run into this one,
+and they will not have to understand the finer points of tuple
+inversion and list sorting to achieve sort-by-value.
 
-    To allow the four basic possibilities of sorting by key/value and in 
-    forward/reverse order, we could add this method:
+To allow the four basic possibilities of sorting by key/value and in
+forward/reverse order, we could add this method::
 
-        (4) sorted_items(by_value=0, reversed=0)
+    (4) sorted_items(by_value=0, reversed=0)
 
-    I believe the most common case would actually be "by_value=1,
-    reversed=1", but the defaults values given here might lead to
-    fewer surprises by users: sorted_items() would be the same as
-    items() followed by sort().
+I believe the most common case would actually be ``by_value=1,
```

Comment:
Here's how it looks like (after I removed the extra backticks at the end)

<img width="816" alt="screen shot 2017-01-23 at 7 14 09 pm" src="https://cloud.githubusercontent.com/assets/5844587/22232750/2da2da36-e1a0-11e6-98bb-fb6751f37931.png">